cryptography/src/bls12381/dkg/types.rs

```diff
@@ -0,0 +1,232 @@
+//! Standard types sent over the wire for [Dealer] shares and [Player] acknowledgements.
+//!
+//! [Dealer]: crate::bls12381::dkg::Dealer
+//! [Player]: crate::bls12381::dkg::Player
+
+use crate::{
+    bls12381::primitives::{group, poly::Public, variant::Variant},
+    PublicKey, Signature, Signer,
+};
+use bytes::{Buf, BufMut};
+use commonware_codec::{varint::UInt, EncodeSize, FixedSize, Read, ReadExt, Write};
+use commonware_utils::quorum;
+
+/// The signature namespace for DKG share acknowledgements.
+pub const ACK_NAMESPACE: &[u8] = b"DKG_ACK";
+
+/// A [Share] sent by a [Dealer] node to a [Player] node.
+///
+/// Contains the [Dealer]'s public commitment to their polynomial and the specific
+/// share calculated for the receiving [Player].
+///
+/// [Share]: group::Share
+/// [Dealer]: crate::bls12381::dkg::Dealer
+/// [Player]: crate::bls12381::dkg::Player
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Share<V: Variant> {
```

Comment:
Once this is in, I can refactor our use of this in `arbiter/player` such that we generate these as dealings and accept directly in `player`.

Reply:
https://github.com/commonwarexyz/monorepo/issues/1816